tests/factories/models.py

```diff
@@ -58,9 +76,36 @@ def newsletters(self, create, extracted, **kwargs):
             for _ in range(extracted):
                 NewsletterFactory(email=self, **kwargs)
 
+    @factory.post_generation
+    def fxa(self, create, extracted, **kwargs):
+        if not create:
+            return
+        if extracted:
+            FirefoxAccountFactory(email=self, **kwargs)
```

Comment:
missing return?

Reply:
Nope -- since this is the `EmailFactory`, all we're doing here is creating the `FirefoxAccount` and associating it with this Email object. In a test, this would work fine:

```
email = EmailFactory()
dbsession.commit()
assert email.fxa ...
```

Unless you're saying that we should avoid the implicit return of `None`, in which case I could make that fix.

Reply:
I get it, `email=self` does it. Sorry for the confusion